app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb

```diff
@@ -342,4 +344,64 @@ def key_pairs
       )
     end
   end
+
+  def service_offerings
+    collector.service_offerings.each do |service_offering|
```

Comment:
What is the order of magnitude of the number of service offerings?  Hundreds/thousands/etc...
Doing API calls in a loop over a collection sucks :/

Reply:
not sure, but there'll be few of those. Only way to make it work nicely is multi-thread + targeted refresh (if possible)